sdk/storage/azblob/container/models.go

```diff
@@ -98,10 +100,56 @@ func (o *GetPropertiesOptions) format() (*generated.ContainerClientGetProperties
 
 // ---------------------------------------------------------------------------------------------------------------------
 
+// ListBlobsDetail indicates what additional information the service should return with each blob.
+type ListBlobsDetail struct {
```

Comment:
This is the type-name from track 1.5, I'm not a huge fan.  What about `ListBlobsInclude`?  Something else?

Reply:
Sure, that's fine